### PR TITLE
Add LLaMAv2-70B text generation

### PR DESCRIPTION
**Pull request by dvarshney-habana**
Update README with performance optimal command for LLaMAv2-70B text generation using deepspeed

# What does this PR do?
Add LLaMAv2-70B text generation commands to README

Fixes # (issue)

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by regisss — COMMENTED** [review]
I left a couple of comments

File: examples/text-generation/README.md [hunk 1]
```diff
@@ -95,6 +95,22 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
...
+--max_new_tokens 4096 \
+--bf16 \
+--use_hpu_graphs \
+--use_kv_cache \
+--batch_size 52 \
+--attn_softmax_bf16 \
+--limit_hpu_graphs \
+--reuse_cache \
+--trim_logits
+
```
Comment (regisss) [inline examples/text-generation/README.md:111]: We have to remove this blank line, otherwise the command is automatically launched when copy-pasting it.

Comment (dvarshney-habana) [reply examples/text-generation/README.md]: good catch, thanks. I have fixed it

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**